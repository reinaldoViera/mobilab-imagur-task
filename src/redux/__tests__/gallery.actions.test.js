import {
    addGalleries
} from '../gallery.actions';
import {
    fakeGallery
} from '../utils';

describe('Gallery actions ', () => {
    it('should add galleries', () => {
        let action = addGalleries(fakeGallery);
        expect(action).toMatchSnapshot();
    });
});