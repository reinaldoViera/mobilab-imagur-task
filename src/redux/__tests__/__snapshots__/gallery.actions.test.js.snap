// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery actions  should add galleries 1`] = `
Object {
  "payload": Object {
    "galleries": Object {
      "09hjKNG": Object {
        "account_id": 17596716,
        "account_url": "theJellificationofStuff",
        "ad_type": 0,
        "ad_url": "",
        "comment_count": 382,
        "cover": "hyrUwx4",
        "cover_height": 408,
        "cover_width": 728,
        "datetime": 1544278865,
        "description": null,
        "downs": 30,
        "favorite": false,
        "favorite_count": 1189,
        "id": "09hjKNG",
        "images": Array [
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 10866332536488,
            "comment_count": null,
            "datetime": 1544277582,
            "description": "A lion gets a attacked by 20 hyenas and he finds himself in some serious trouble",
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/hyrUwx4.gifv",
            "has_sound": false,
            "height": 408,
            "hls": "https://i.imgur.com/hyrUwx4.m3u8",
            "id": "hyrUwx4",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "https://i.imgur.com/hyrUwx4.gif",
            "looping": true,
            "mp4": "https://i.imgur.com/hyrUwx4.mp4",
            "mp4_size": 2293104,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 19744296,
            "tags": Array [],
            "title": null,
            "type": "image/gif",
            "ups": null,
            "views": 550353,
            "vote": null,
            "width": 728,
          },
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 6705571128755,
            "comment_count": null,
            "datetime": 1544277820,
            "description": "It starts to get pretty bad for him",
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/aSyTlJg.gifv",
            "has_sound": false,
            "height": 408,
            "hls": "https://i.imgur.com/aSyTlJg.m3u8",
            "id": "aSyTlJg",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "http://i.imgur.com/aSyTlJgh.gif",
            "looping": true,
            "mp4": "https://i.imgur.com/aSyTlJg.mp4",
            "mp4_size": 3079870,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 23881855,
            "tags": Array [],
            "title": null,
            "type": "image/gif",
            "ups": null,
            "views": 280781,
            "vote": null,
            "width": 728,
          },
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 7854894827676,
            "comment_count": null,
            "datetime": 1544278406,
            "description": "He calls out to his bro for help",
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/PBrVVXc.gifv",
            "has_sound": false,
            "height": 408,
            "hls": "https://i.imgur.com/PBrVVXc.m3u8",
            "id": "PBrVVXc",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "http://i.imgur.com/PBrVVXch.gif",
            "looping": true,
            "mp4": "https://i.imgur.com/PBrVVXc.mp4",
            "mp4_size": 2892601,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 23777637,
            "tags": Array [],
            "title": null,
            "type": "image/gif",
            "ups": null,
            "views": 330348,
            "vote": null,
            "width": 728,
          },
        ],
        "images_count": 5,
        "in_gallery": true,
        "in_most_viral": true,
        "include_album_ads": false,
        "is_ad": false,
        "is_album": true,
        "layout": "blog",
        "link": "https://imgur.com/a/09hjKNG",
        "nsfw": false,
        "points": 3828,
        "privacy": "public",
        "score": 3875,
        "section": "",
        "tags": Array [
          Object {
            "accent": "8472BD",
            "background_hash": "4kmYoey",
            "background_is_animated": false,
            "description": "neat and amazing",
            "description_annotations": Object {},
            "display_name": "awesome",
            "followers": 562155,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "awesome",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 988368,
          },
          Object {
            "accent": "50535A",
            "background_hash": "A2MU7EC",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "hyena",
            "followers": 103,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "hyena",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 122,
          },
          Object {
            "accent": "5A3D74",
            "background_hash": "CXErE2G",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "nature is awesome",
            "followers": 3289,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "nature_is_awesome",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 3689,
          },
          Object {
            "accent": "276860",
            "background_hash": "1eQyuB1",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "damnnatureyouscary",
            "followers": 137,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "damnnatureyouscary",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 128,
          },
        ],
        "title": "Lion gets attacked by 20 Hyenas",
        "topic": "No Topic",
        "topic_id": 29,
        "ups": 3858,
        "views": 94174,
        "vote": null,
      },
      "5ObAM69": Object {
        "account_id": 57469749,
        "account_url": "TheTaterNutz",
        "ad_type": 0,
        "ad_url": "",
        "comment_count": 125,
        "cover": "vW24zCQ",
        "cover_height": 240,
        "cover_width": 245,
        "datetime": 1544288307,
        "description": null,
        "downs": 13,
        "favorite": false,
        "favorite_count": 93,
        "id": "5ObAM69",
        "images": Array [
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 161158703706,
            "comment_count": null,
            "datetime": 1544281858,
            "description": null,
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/vW24zCQ.gifv",
            "has_sound": false,
            "height": 240,
            "hls": "https://i.imgur.com/vW24zCQ.m3u8",
            "id": "vW24zCQ",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "https://i.imgur.com/vW24zCQ.gif",
            "looping": true,
            "mp4": "https://i.imgur.com/vW24zCQ.mp4",
            "mp4_size": 105367,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 1672083,
            "tags": Array [],
            "title": null,
            "type": "image/gif",
            "ups": null,
            "views": 96382,
            "vote": null,
            "width": 245,
          },
        ],
        "images_count": 1,
        "in_gallery": true,
        "in_most_viral": true,
        "include_album_ads": false,
        "is_ad": false,
        "is_album": true,
        "layout": "blog",
        "link": "https://imgur.com/a/5ObAM69",
        "nsfw": false,
        "points": 1234,
        "privacy": "hidden",
        "score": 1245,
        "section": "",
        "tags": Array [],
        "title": "Brother's GF of two weeks wants a list of all of his passwords \\"to build their relationship on trust\\"",
        "topic": "No Topic",
        "topic_id": 29,
        "ups": 1247,
        "views": 23011,
        "vote": null,
      },
      "LnYks6g": Object {
        "account_id": 94293853,
        "account_url": "Thisisminemind",
        "ad_type": 0,
        "ad_url": "",
        "comment_count": 108,
        "cover": "FIQH2hA",
        "cover_height": 268,
        "cover_width": 268,
        "datetime": 1544287669,
        "description": null,
        "downs": 66,
        "favorite": false,
        "favorite_count": 223,
        "id": "LnYks6g",
        "images": Array [
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 85002062535,
            "comment_count": null,
            "datetime": 1544287633,
            "description": null,
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/FIQH2hA.gifv",
            "has_sound": false,
            "height": 268,
            "hls": "https://i.imgur.com/FIQH2hA.m3u8",
            "id": "FIQH2hA",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "https://i.imgur.com/FIQH2hA.gif",
            "looping": true,
            "mp4": "https://i.imgur.com/FIQH2hA.mp4",
            "mp4_size": 66060,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 678015,
            "tags": Array [],
            "title": null,
            "type": "image/gif",
            "ups": null,
            "views": 125369,
            "vote": null,
            "width": 268,
          },
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 156041502768,
            "comment_count": null,
            "datetime": 1544287638,
            "description": null,
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/NmUyiZw.gifv",
            "has_sound": false,
            "height": 268,
            "hls": "https://i.imgur.com/NmUyiZw.m3u8",
            "id": "NmUyiZw",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "https://i.imgur.com/NmUyiZw.gif",
            "looping": true,
            "mp4": "https://i.imgur.com/NmUyiZw.mp4",
            "mp4_size": 196412,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 2114928,
            "tags": Array [],
            "title": null,
            "type": "image/gif",
            "ups": null,
            "views": 73781,
            "vote": null,
            "width": 268,
          },
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 75296090641,
            "comment_count": null,
            "datetime": 1544287642,
            "description": null,
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/u6UR5u6.gifv",
            "has_sound": false,
            "height": 268,
            "hls": "https://i.imgur.com/u6UR5u6.m3u8",
            "id": "u6UR5u6",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "https://i.imgur.com/u6UR5u6.gif",
            "looping": true,
            "mp4": "https://i.imgur.com/u6UR5u6.mp4",
            "mp4_size": 112857,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 1274671,
            "tags": Array [],
            "title": null,
            "type": "image/gif",
            "ups": null,
            "views": 59071,
            "vote": null,
            "width": 268,
          },
        ],
        "images_count": 4,
        "in_gallery": true,
        "in_most_viral": true,
        "include_album_ads": false,
        "is_ad": false,
        "is_album": true,
        "layout": "blog",
        "link": "https://imgur.com/a/LnYks6g",
        "nsfw": false,
        "points": 1260,
        "privacy": "public",
        "score": 1277,
        "section": "",
        "tags": Array [
          Object {
            "accent": "633875",
            "background_hash": "9r1qCDq",
            "background_is_animated": false,
            "description": "LOLs, ROFLs, LMAOs",
            "description_annotations": Object {},
            "display_name": "funny",
            "followers": 1765087,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "funny",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 3383669,
          },
          Object {
            "accent": "8AA816",
            "background_hash": "oAtvuB2",
            "background_is_animated": false,
            "description": "weird, wild, why?",
            "description_annotations": Object {},
            "display_name": "wtf",
            "followers": 680684,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "wtf",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 83890,
          },
          Object {
            "accent": "60AEBB",
            "background_hash": "avRBRpN",
            "background_is_animated": false,
            "description": "cute and adorable",
            "description_annotations": Object {},
            "display_name": "aww",
            "followers": 942359,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "aww",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 998616,
          },
          Object {
            "accent": "444C92",
            "background_hash": "gbuk2AN",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "lol",
            "followers": 39661,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "lol",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 52890,
          },
          Object {
            "accent": "2084B0",
            "background_hash": "JdeWoaA",
            "background_is_animated": false,
            "description": "start your binge",
            "description_annotations": Object {},
            "display_name": "movies and tv",
            "followers": 624817,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "movies_and_tv",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 127577,
          },
        ],
        "title": "Comedy comes from love",
        "topic": "No Topic",
        "topic_id": 29,
        "ups": 1326,
        "views": 34691,
        "vote": null,
      },
      "MUNNfUj": Object {
        "account_id": 3123245,
        "account_url": "deafkitten",
        "ad_type": 0,
        "ad_url": "",
        "comment_count": 43,
        "cover": "iC4xKxn",
        "cover_height": 480,
        "cover_width": 854,
        "datetime": 1544285956,
        "description": null,
        "downs": 30,
        "favorite": false,
        "favorite_count": 157,
        "id": "MUNNfUj",
        "images": Array [
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 112540921020,
            "comment_count": null,
            "datetime": 1544285953,
            "description": "fun stuff.",
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/iC4xKxn.gifv",
            "has_sound": false,
            "height": 480,
            "hls": "https://i.imgur.com/iC4xKxn.m3u8",
            "id": "iC4xKxn",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "https://i.imgur.com/iC4xKxn.mp4",
            "mp4": "https://i.imgur.com/iC4xKxn.mp4",
            "mp4_size": 1035945,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 1035945,
            "tags": Array [],
            "title": null,
            "type": "video/mp4",
            "ups": null,
            "views": 108636,
            "vote": null,
            "width": 854,
          },
        ],
        "images_count": 1,
        "in_gallery": true,
        "in_most_viral": true,
        "include_album_ads": false,
        "is_ad": false,
        "is_album": true,
        "layout": "blog",
        "link": "https://imgur.com/a/MUNNfUj",
        "nsfw": false,
        "points": 1343,
        "privacy": "hidden",
        "score": 1363,
        "section": "",
        "tags": Array [],
        "title": "Seeing a break up post in usersub longer than a few paragraphs ",
        "topic": "No Topic",
        "topic_id": 29,
        "ups": 1373,
        "views": 41485,
        "vote": null,
      },
      "UraDtCx": Object {
        "account_id": 66894193,
        "account_url": "Yoda1PositivityStar",
        "ad_type": 0,
        "ad_url": "",
        "comment_count": 379,
        "cover": "D8iB0DE",
        "cover_height": 518,
        "cover_width": 518,
        "datetime": 1544280460,
        "description": null,
        "downs": 31,
        "favorite": false,
        "favorite_count": 1326,
        "id": "UraDtCx",
        "images": Array [
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 23290393751836,
            "comment_count": null,
            "datetime": 1544280456,
            "description": null,
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/D8iB0DE.gifv",
            "has_sound": false,
            "height": 518,
            "hls": "https://i.imgur.com/D8iB0DE.m3u8",
            "id": "D8iB0DE",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "http://i.imgur.com/D8iB0DEh.gif",
            "looping": true,
            "mp4": "https://i.imgur.com/D8iB0DE.mp4",
            "mp4_size": 3720271,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 50664994,
            "tags": Array [],
            "title": null,
            "type": "image/gif",
            "ups": null,
            "views": 459694,
            "vote": null,
            "width": 518,
          },
        ],
        "images_count": 1,
        "in_gallery": true,
        "in_most_viral": true,
        "include_album_ads": false,
        "is_ad": false,
        "is_album": true,
        "layout": "blog",
        "link": "https://imgur.com/a/UraDtCx",
        "nsfw": false,
        "points": 5246,
        "privacy": "hidden",
        "score": 5290,
        "section": "",
        "tags": Array [
          Object {
            "accent": "8472BD",
            "background_hash": "4kmYoey",
            "background_is_animated": false,
            "description": "neat and amazing",
            "description_annotations": Object {},
            "display_name": "awesome",
            "followers": 562155,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "awesome",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 988369,
          },
          Object {
            "accent": "3218CE",
            "background_hash": "jOsus41",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "neat",
            "followers": 8258,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "neat",
            "thumbnail_hash": "jOsus41",
            "thumbnail_is_animated": false,
            "total_items": 6527,
          },
          Object {
            "accent": null,
            "background_hash": "xVuIbkP",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "snow",
            "followers": 4928,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "snow",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 10193,
          },
          Object {
            "accent": "51535A",
            "background_hash": "7bemn9H",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "timelapse",
            "followers": 589,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "timelapse",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 1107,
          },
        ],
        "title": "Crazy how nature do that ",
        "topic": "No Topic",
        "topic_id": 29,
        "ups": 5277,
        "views": 89858,
        "vote": null,
      },
      "uRBau9B": Object {
        "account_id": 14836716,
        "account_url": "MrPuckett",
        "ad_type": 0,
        "ad_url": "",
        "comment_count": 34,
        "cover": "r68tyB7",
        "cover_height": 720,
        "cover_width": 720,
        "datetime": 1544286098,
        "description": null,
        "downs": 36,
        "favorite": false,
        "favorite_count": 318,
        "id": "uRBau9B",
        "images": Array [
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": true,
            "bandwidth": 209253777636,
            "comment_count": null,
            "datetime": 1544286095,
            "description": null,
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "gifv": "https://i.imgur.com/r68tyB7.gifv",
            "has_sound": false,
            "height": 720,
            "hls": "https://i.imgur.com/r68tyB7.m3u8",
            "id": "r68tyB7",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "https://i.imgur.com/r68tyB7.mp4",
            "mp4": "https://i.imgur.com/r68tyB7.mp4",
            "mp4_size": 1595372,
            "nsfw": null,
            "points": null,
            "processing": Object {
              "status": "completed",
            },
            "score": null,
            "section": null,
            "size": 1595372,
            "tags": Array [],
            "title": null,
            "type": "video/mp4",
            "ups": null,
            "views": 131163,
            "vote": null,
            "width": 720,
          },
        ],
        "images_count": 1,
        "in_gallery": true,
        "in_most_viral": true,
        "include_album_ads": false,
        "is_ad": false,
        "is_album": true,
        "layout": "blog",
        "link": "https://imgur.com/a/uRBau9B",
        "nsfw": false,
        "points": 1353,
        "privacy": "public",
        "score": 1375,
        "section": "",
        "tags": Array [
          Object {
            "accent": "633875",
            "background_hash": "9r1qCDq",
            "background_is_animated": false,
            "description": "LOLs, ROFLs, LMAOs",
            "description_annotations": Object {},
            "display_name": "funny",
            "followers": 1765064,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "funny",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 3383664,
          },
          Object {
            "accent": "8472BD",
            "background_hash": "4kmYoey",
            "background_is_animated": false,
            "description": "neat and amazing",
            "description_annotations": Object {},
            "display_name": "awesome",
            "followers": 562155,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "awesome",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 988368,
          },
          Object {
            "accent": "93BDFF",
            "background_hash": "DDUrxGU",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "dogs are the best people",
            "followers": 28993,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "dogs_are_the_best_people",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 19648,
          },
        ],
        "title": "Just a regular Saturday",
        "topic": "No Topic",
        "topic_id": 29,
        "ups": 1389,
        "views": 45488,
        "vote": null,
      },
      "whGYa7X": Object {
        "account_id": 97869609,
        "account_url": "IncredibleMsDee",
        "ad_type": 0,
        "ad_url": "",
        "comment_count": 347,
        "cover": "dqM44KR",
        "cover_height": 1564,
        "cover_width": 1564,
        "datetime": 1544277883,
        "description": null,
        "downs": 221,
        "favorite": false,
        "favorite_count": 344,
        "id": "whGYa7X",
        "images": Array [
          Object {
            "account_id": null,
            "account_url": null,
            "ad_type": 0,
            "ad_url": "",
            "animated": false,
            "bandwidth": 77506830508,
            "comment_count": null,
            "datetime": 1544277843,
            "description": null,
            "downs": null,
            "favorite": false,
            "favorite_count": null,
            "has_sound": false,
            "height": 1564,
            "id": "dqM44KR",
            "in_gallery": false,
            "in_most_viral": false,
            "is_ad": false,
            "link": "https://i.imgur.com/dqM44KR.jpg",
            "nsfw": null,
            "points": null,
            "score": null,
            "section": null,
            "size": 780422,
            "tags": Array [],
            "title": null,
            "type": "image/jpeg",
            "ups": null,
            "views": 99314,
            "vote": null,
            "width": 1564,
          },
        ],
        "images_count": 1,
        "in_gallery": true,
        "in_most_viral": true,
        "include_album_ads": false,
        "is_ad": false,
        "is_album": true,
        "layout": "blog",
        "link": "https://imgur.com/a/whGYa7X",
        "nsfw": false,
        "points": 4481,
        "privacy": "public",
        "score": 4534,
        "section": "",
        "tags": Array [
          Object {
            "accent": "4D549A",
            "background_hash": "ILiOaNX",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "weightloss",
            "followers": 986,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "weightloss",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 1336,
          },
          Object {
            "accent": "5B62A5",
            "background_hash": "f8B0kEw",
            "background_is_animated": false,
            "description": "",
            "description_annotations": Object {},
            "display_name": "weightlossjourney",
            "followers": 144,
            "following": false,
            "is_promoted": false,
            "logo_destination_url": null,
            "logo_hash": null,
            "name": "weightlossjourney",
            "thumbnail_hash": null,
            "thumbnail_is_animated": false,
            "total_items": 92,
          },
        ],
        "title": "204lbs to 145 lbs in one year",
        "topic": "No Topic",
        "topic_id": 29,
        "ups": 4702,
        "views": 108230,
        "vote": null,
      },
    },
    "list": Array [
      "UraDtCx",
      "5ObAM69",
      "LnYks6g",
      "uRBau9B",
      "MUNNfUj",
      "whGYa7X",
      "09hjKNG",
    ],
    "replace": false,
  },
  "type": "Add galleries",
}
`;
